students/kegan/session07/test_html_render.py

```diff
@@ -0,0 +1,208 @@
+"""
+Kathryn Egan
+
+Suite of tests for components of html_render.py
+"""
+from html_render import *
+
+
+def test_element_init():
+    element = Element()
+    assert element.content == []
+    element = Element('content')
+    assert element.content == ['content']
+    element = Element('')
+    assert element.content == ['']
+
+
+def test_append():
+    element = Element()
+    element.append('content')
+    element.append('more content')
+    assert element.content == ['content', 'more content']
+
+
+# def test_render_no_indent():
+#     html = Html()
+#     html.append(Body('body content'))
+#     html.append(P())
+#     temp = 'html_test.html'
+#     with open(temp, 'w') as f:
+#         html.render(f)
+#     with open(temp, 'r') as f:
+#         assert f.read() == '<html><body>body content</body><p></p></html>'
+
+
+def prepare(elements):
```

Comment:
this makes me a little nervous -- kinda fixed results expected -- but great instinct on automating the testing process. And I guess if you do change anything, you can update this one function :-)